lib/ui/views/app_selector/app_selector_view.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:flutter/material.dart';
+import 'package:flutter/material.dart' hide SearchBar;
```

Comment:
Why is SearchBar being hidden?

Reply:
Newer Flutter version have SearchBar which conflicts with already using SearchBar.